### PR TITLE
Add rule elements to Feral Sense and You don't smell right feats

### LINKED ISSUE #15104: Request for "feral senses" and "You don't smell right"  automation
Would it be possible to automate the "feral senses" and "You don't smell right"  feats from the werecreature archtype? The senses don't apply when entering were creature forms and You don't smell right doesn't add the sense or increase any existing scent range.

### PR DESCRIPTION
**Pull request by ammalagonc**
Closes #15104

### CODE REVIEW

File: packs/feats/you-dont-smell-right.json [hunk 1]
```diff
@@ -28,7 +28,33 @@
             "remaster": true,
             "title": "Pathfinder Howl of the Wild"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "Smell disguise",
```
Comment (pedrogrullada) [inline packs/feats/you-dont-smell-right.json:35]: Don't forget to localise this. I'd maybe go with something more descriptive like "Seek to determine if a creature is disguised" or something along those lines

Comment (ammalagonc) [reply packs/feats/you-dont-smell-right.json]: Added localization and expanded on the description.

File: packs/feats/feral-senses.json [hunk 2]
```diff
@@ -28,7 +28,18 @@
...
+            {
+                "acuity": "imprecise",
+                "key": "Sense",
+                "range": 30,
+                "selector": "scent"
+            },
+            {
+                "key": "Sense",
+                "selector": "low-light-vision"
+            }
```
Comment (TikaelSol) [inline packs/feats/feral-senses.json:41]: These should be predicated on being in hybrid or animal form.

File: packs/feats/you-dont-smell-right.json [hunk 3]
```diff
@@ -28,7 +28,33 @@
...
+            },
+            {
+                "acuity": "imprecise",
+                "key": "Sense",
+                "predicate": [
+                    "feat:feral-senses"
+                ],
+                "range": 60,
+                "selector": "scent"
+            },
```
Comment (TikaelSol) [inline packs/feats/you-dont-smell-right.json:47]: This should be an unpredicated 30, and a predicated 60 on having the feat and being in hybrid or animal shape (when feral senses would be active)

File: packs/feats/you-dont-smell-right.json [hunk 4]
```diff
@@ -28,7 +28,41 @@
...
+                        "and": [
+                            "self:effect:change-shape-werecreature",
+                            "feat:feral-senses"
+                        ]
+                    }
+                ],
+                "range": 60,
+                "selector": "scent"
+            },
+            {
```
Comment (TikaelSol) [inline packs/feats/you-dont-smell-right.json:56]:
```suggestion
            },
            {
                "acuity": "imprecise",
                "key": "Sense",
                "predicate": [
                    {
                        "and": [
                            "self:effect:change-shape-werecreature",
                            "feat:feral-senses"
                        ]
                    }
                ],
                "range": 60,
                "selector": "scent"
            },
            {
                "acuity": "imprecise",
                "key": "Sense",
                "range": 30,
                "selector": "scent"
            },
            {
```

**Pull request merged**